code/modules/uplink/uplink_items/device_tools.dm

```diff
@@ -61,13 +61,13 @@
 	cost = 1
 	surplus = 20
 
-/* /datum/uplink_item/device_tools/briefcase_launchpad // SKYRAT EDIT REMOVAL
+/datum/uplink_item/device_tools/briefcase_launchpad
 	name = "Briefcase Launchpad"
 	desc = "A briefcase containing a launchpad, a device able to teleport items and people to and from targets up to eight tiles away from the briefcase. \
 			Also includes a remote control, disguised as an ordinary folder. Touch the briefcase with the remote to link it."
 	surplus = 0
 	item = /obj/item/storage/briefcase/launchpad
-	cost = 6 */
+	cost = 6
```

Comment:
```suggestion
	cost = 11
```
Trivialises almost all steal objectives, gives you an easy way to snatch all of the head lockers with zero risk, this is several orders of magnitude more robust with our population size than it would be on more populated servers with 80+ people running around.
11TC for every head locker 100% risk-free on most maps is still very cheap.